### ISSUE TITLE
【EESS】New submissions for Wed, 30 Aug 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#578]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Unsupervised Active Learning: Optimizing Labeling Cost-Effectiveness for  Automatic Speech Recognition
 - **Authors:** Zhisheng Zheng, Ziyang Ma, Yu Wang, Xie Chen
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2308.14814
 - **Pdf link:** https://arxiv.org/pdf/2308.14814
 - **Abstract**
 In recent years, speech-based self-supervised learning (SSL) has made significant progress in various tasks, including automatic speech recognition (ASR). An ASR model with decent performance can be realized by fine-tuning an SSL model with a small fraction of labeled data. Reducing the demand for labeled data is always of great practical value. In this paper, we further extend the use of SSL to cut down labeling costs with active learning. Three types of units on different granularities are derived from speech signals in an unsupervised way, and their effects are compared by applying a contrastive data selection method. The experimental results show that our proposed data selection framework can effectively improve the word error rate (WER) by more than 11% with the same amount of labeled data, or halve the labeling cost while maintaining the same WER, compared to random selection.
### Uncertainty Aware Training to Improve Deep Learning Model Calibration  for Classification of Cardiac MR Images
 - **Authors:** Tareen Dawood, Chen Chen, Baldeep S. Sidhua, Bram Ruijsink, Justin Goulda, Bradley Porter, Mark K. Elliott, Vishal Mehta, Christopher A. Rinaldi, Esther Puyol-Anton, Reza Razavi, Andrew P. King
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2308.15141
 - **Pdf link:** https://arxiv.org/pdf/2308.15141
 - **Abstract**
 Quantifying uncertainty of predictions has been identified as one way to develop more trustworthy artificial intelligence (AI) models beyond conventional reporting of performance metrics. When considering their role in a clinical decision support setting, AI classification models should ideally avoid confident wrong predictions and maximise the confidence of correct predictions. Models that do this are said to be well-calibrated with regard to confidence. However, relatively little attention has been paid to how to improve calibration when training these models, i.e., to make the training strategy uncertainty-aware. In this work we evaluate three novel uncertainty-aware training strategies comparing against two state-of-the-art approaches. We analyse performance on two different clinical applications: cardiac resynchronisation therapy (CRT) response prediction and coronary artery disease (CAD) diagnosis from cardiac magnetic resonance (CMR) images. The best-performing model in terms of both classification accuracy and the most common calibration measure, expected calibration error (ECE) was the Confidence Weight method, a novel approach that weights the loss of samples to explicitly penalise confident incorrect predictions. The method reduced the ECE by 17% for CRT response prediction and by 22% for CAD diagnosis when compared to a baseline classifier in which no uncertainty-aware strategy was included. In both applications, as well as reducing the ECE there was a slight increase in accuracy from 69% to 70% and 70% to 72% for CRT response prediction and CAD diagnosis respectively. However, our analysis showed a lack of consistency in terms of optimal models when using different calibration measures. This indicates the need for careful consideration of performance metrics when training and selecting models for complex high-risk applications in healthcare.
### Let There Be Sound: Reconstructing High Quality Speech from Silent  Videos
 - **Authors:** Ji-Hoon Kim, Jaehun Kim, Joon Son Chung
 - **Subjects:** Audio and Speech Processing (eess.AS); Artificial Intelligence (cs.AI); Machine Learning (cs.LG); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2308.15256
 - **Pdf link:** https://arxiv.org/pdf/2308.15256
 - **Abstract**
 The goal of this work is to reconstruct high quality speech from lip motions alone, a task also known as lip-to-speech. A key challenge of lip-to-speech systems is the one-to-many mapping caused by (1) the existence of homophenes and (2) multiple speech variations, resulting in a mispronounced and over-smoothed speech. In this paper, we propose a novel lip-to-speech system that significantly improves the generation quality by alleviating the one-to-many mapping problem from multiple perspectives. Specifically, we incorporate (1) self-supervised speech representations to disambiguate homophenes, and (2) acoustic variance information to model diverse speech styles. Additionally, to better solve the aforementioned problem, we employ a flow based post-net which captures and refines the details of the generated speech. We perform extensive experiments and demonstrate that our method achieves the generation quality close to that of real human utterance, outperforming existing methods in terms of speech naturalness and intelligibility by a large margin. Synthesised samples are available at the anonymous demo page: https://mm.kaist.ac.kr/projects/LTBS.
### Towards quantitative precision for ECG analysis: Leveraging state space  models, self-supervision and patient metadata
 - **Authors:** Temesgen Mehari, Nils Strodthoff
 - **Subjects:** Signal Processing (eess.SP); Machine Learning (cs.LG); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2308.15291
 - **Pdf link:** https://arxiv.org/pdf/2308.15291
 - **Abstract**
 Deep learning has emerged as the preferred modeling approach for automatic ECG analysis. In this study, we investigate three elements aimed at improving the quantitative accuracy of such systems. These components consistently enhance performance beyond the existing state-of-the-art, which is predominantly based on convolutional models. Firstly, we explore more expressive architectures by exploiting structured state space models (SSMs). These models have shown promise in capturing long-term dependencies in time series data. By incorporating SSMs into our approach, we not only achieve better performance, but also gain insights into long-standing questions in the field. Specifically, for standard diagnostic tasks, we find no advantage in using higher sampling rates such as 500Hz compared to 100Hz. Similarly, extending the input size of the model beyond 3 seconds does not lead to significant improvements. Secondly, we demonstrate that self-supervised learning using contrastive predictive coding can further improve the performance of SSMs. By leveraging self-supervision, we enable the model to learn more robust and representative features, leading to improved analysis accuracy. Lastly, we depart from synthetic benchmarking scenarios and incorporate basic demographic metadata alongside the ECG signal as input. This inclusion of patient metadata departs from the conventional practice of relying solely on the signal itself. Remarkably, this addition consistently yields positive effects on predictive performance. We firmly believe that all three components should be considered when developing next-generation ECG analysis algorithms.
### A Scalable Approach for Analysing Multi-Agent Systems with Heterogeneous  Stochastic Packet Loss
 - **Authors:** Christian Hespe, Herbert Werner
 - **Subjects:** Systems and Control (eess.SY); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2308.15360
 - **Pdf link:** https://arxiv.org/pdf/2308.15360
 - **Abstract**
 An important aspect in jointly analysing networked control systems and their communication is to model the networking in a sufficiently rich but at the same time mathematically tractable way. As such, this paper improves on a recently proposed scalable approach for analysing multi-agent systems with stochastic packet loss by allowing for heterogeneous transmission probabilities and temporal correlation in the communication model. The key idea is to consider the transmission probabilities as uncertain, which facilitates the use of tools from robust control. Due to being formulated in terms of linear matrix inequalities that grow linearly with the number of agents, the result is applicable to very large multi-agent systems, which is demonstrated by numerical simulations with up to 10000 agents.
### Meta-learning for model-reference data-driven control
 - **Authors:** Riccardo Busetto, Valentina Breschi, Simone Formentin
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2308.15458
 - **Pdf link:** https://arxiv.org/pdf/2308.15458
 - **Abstract**
 One-shot direct model-reference control design techniques, like the Virtual Reference Feedback Tuning (VRFT) approach, offer time-saving solutions for the calibration of fixed-structure controllers for dynamic systems. Nonetheless, such methods are known to be highly sensitive to the quality of the available data, often requiring long and costly experiments to attain acceptable closed-loop performance. These features might prevent the widespread adoption of such techniques, especially in low-data regimes. In this paper, we argue that the inherent similarity of many industrially relevant systems may come at hand, offering additional information from plants that are similar (yet not equal) to the system one aims to control. Assuming that this supplementary information is available, we propose a novel, direct design approach that leverages the data from similar plants, the knowledge of controllers calibrated on them, and the corresponding closed-loop performance to enhance model-reference control design. More specifically, by constructing the new controller as a combination of the available ones, our approach exploits all the available priors following a meta-learning philosophy, while ensuring non-decreasing performance. An extensive numerical analysis supports our claims, highlighting the effectiveness of the proposed method in achieving performance comparable to iterative approaches, while at the same time retaining the efficiency of one-shot direct data-driven methods like VRFT.
